Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Magazines/rifle.yml

```
@@ -62,16 +64,18 @@
 
 - type: entity
   id: MagazineRifleIncendiary
-  name: "magazine (.20 rifle incendiary)"
+  name: "magazine (.223 rifle incendiary)"
   parent: MagazineRifle
+  description: The standard magazine for NT combat rifles, holds 25 rounds. This type contains red-tipped incendiary ammunition that burns the target from the inside.
   components:
   - type: BallisticAmmoProvider
     proto: CartridgeRifleIncendiary
 
 - type: entity
   id: MagazineRiflePractice
-  name: "magazine (.20 rifle practice)"
+  name: "magazine (.223 rifle practice)"
   parent: BaseMagazineRifle
+  description: The standard magazine for NT combat rifles, holds 25 rounds. This type contains practice ammo with blue rubber tips, denoted by a blue strip around the magazine.
```

Obvious things, plus rubber != practice.

```suggestion
  description: The standard magazine for NT combat rifles, holds 25 rounds. This type should contain practice ammo.
```